lightly_studio/src/lightly_studio/core/add_videos.py

```diff
@@ -19,7 +20,7 @@
 from sqlmodel import Session
 from tqdm import tqdm
 
-from lightly_studio.core import logging
+from lightly_studio.core import logging as core_logging
```

Comment:
It would be better to rename the old file. Can you call it e.g. `loading_log.py`?